src/main/java/fr/techad/sonar/GerritPostJob.java

```diff
@@ -183,8 +181,8 @@ protected ReviewLineComment issueToComment(Issue issue) {
 		ReviewLineComment result = new ReviewLineComment();
 
 		result.setLine(issue.line());
-		result.setMessage(String.format(ISSUE_FORMAT, issue.isNew(), StringUtils.capitalize(issue.ruleKey().toString()),
```

Comment:
Technically, we're losing the StringUtils.capitalize() here

I hope it's OK. The problem is that sonar server lookups are case sensitive so capitalizing it makes sonar unable to find it back.